app.js

```diff
@@ -27,10 +28,18 @@ let mainWindow = null;
 let previewWindow = null;
 
 function createTourWindow() {
+  // Easier to work with when centering tour
+  // window on primary display.
+  const width = 700;
```

Comment:
Also, why do we need to declare `width` and `height` here?

Reply:
I use the `width` and `height` variable in two places, for the **tour window** and for **centering the tour window on your display/monitor**. 

So if you ever have to change the `width` and `height` you only have to change it one place and the tour window will still be centered. I just thought that would make it easier, but I can of course hard code the number if you like?

Reply:
Oh I see it now, sorry 😛